### DISCUSSION TITLE
Only show 3rd party OAuth buttons in UI

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#2318] [Questions]
Is it possible to disable the email/password log in boxes in the Supabase UI?  I'm trying to only have a single OAuth login available, but can't find where I could disable the email/password

**Comment by harshcut** [comment]
There is no way to disable or remove the password based authentication method from the `Auth` component. You have to create it on your own. I think the option (`onlyThirdPartyProviders`) to hide the password based authentication method should be added (@MildTomato).

![image](https://user-images.githubusercontent.com/86551248/125199223-d0b3c400-e282-11eb-8006-71d970692305.png)

**Reply by MildTomato** [reply]
@harshcut good idea! Yep, I agree, I think this would be useful.
I'll create an issue for this 👍

**Reply by MildTomato** [reply]
https://github.com/supabase/ui/issues/244
